docs/3.0.0-beta.0/guides/public-assets.md

```diff
@@ -0,0 +1,19 @@
+# Public Assets
+
+See the [public assets concepts](../concepts/concepts.md#public-assets) for details.
+
+Because an API may need to serve static assets, every new Strapi project includes by default, a folder named `/public`. Any file located in this directory is accessible if the request's path doesn't match any other defined route and if it matches a public file name.
```

Comment:
Add a note that it is **not** recommended to use the public folder for serving a frontend, should be used for assets